code/Control/UserDefinedFormController.php

```diff
@@ -429,9 +430,18 @@ public function process($data, $form)
                     }
 
                     $email->setBody($body);
-                    $email->sendPlain();
+
+                    try {
+                        $email->sendPlain();
+                    } catch (Exception $e) {
+                        Injector::inst()->get(LoggerInterface::class)->warning($e);
```

Comment:
I think a failed email send should be an error or emergency, not a warning.

Reply:
@dhensby upgraded to an `error` now